.github/workflows/build-python.yml

```diff
@@ -209,4 +209,59 @@ jobs:
           poetry new ./install-test
```

Comment:
could you rename this step to "Create new project and install **server** wheel **(Poetry)**"

Reply:
Done